builtin/logical/plugin/backend.go

```diff
@@ -0,0 +1,39 @@
+package plugin
```

Comment:
I think there should be a way to restart a killed plugin process. This will probably have to be at the client or logical/plugin/backend.go layer. Checkout how the database plugins do it here https://github.com/hashicorp/vault/blob/2feb9428e7bcc0666eda66fde6b8007d39bfd011/builtin/logical/database/backend.go#L162